lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text"><%- gettext("Sign in") %></span>
+        </h2>
+    </div>
+
+    <p><%- _.sprintf( gettext("Would you like to sign in using your %(providerName)s credentials?"), { providerName: hintedProvider.name } ) %></p>
+
+    <button class="action action-primary action-update proceed-button button-<%- hintedProvider.id %> hinted-login-<%- hintedProvider.id %>">
+        <div class="icon fa <%- hintedProvider.iconClass %>" aria-hidden="true"></div>
```

Comment:
Visual Feedback: Please remove this icon for all cases where there is no formal branding (i.e. only Google, Facebook, Twitter). Its more manageable to **not** have to pick a Font Awesome icon that may or may not fit the bill of the provider.